<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiListScaledImages</class>
 <widget class="QWidget" name="GuiListScaledImages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiListScaledImages</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Scaled Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_read">
          <property name="text">
           <string>Read</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Kernel Mask</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name Kernel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_nameKernel">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>kernel3x3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_createkernel">
          <property name="text">
           <string>Create Kernel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_savekernel">
            <property name="text">
             <string>Save Kernel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_savelistkernel">
            <property name="text">
             <string>Save List Kernel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="text_kernelmask">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_showimage">
            <property name="text">
             <string>Show Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_scaledfactor">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>15</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_showlistimage">
          <property name="text">
           <string>Show List Images</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>CNN Pooling</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Pooing Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spin_poolingsize"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btn_maxpooling">
          <property name="text">
           <string>Max Pooling</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="text_pooling"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
